hydra-main/src/main/java/com/addthis/hydra/job/spawn/balancer/HostScore.java

```diff
@@ -32,4 +34,8 @@ public double getOverallScore() {
     public double getScoreValue(boolean diskSpace) {
```

Comment:
No. But this is just preliminary work before trying to fix everything wrong with this code.